### PR TITLE
Added repository, mapper, services and dto. Tested and looks good to use

### PR DESCRIPTION
**Pull request by Zel1oy**

### CODE REVIEW

File: src/main/java/com/example/onlinebookstore/service/BookService.java [hunk 1]
```diff
@@ -1,10 +1,13 @@
...
 import java.util.List;
 
 public interface BookService {
-    Book save(Book book);
+    BookDto save(CreateBookRequestDto book);
 
-    List<Book> findAll();
+    List<BookDto> findAll();
+
+    BookDto getBookById(Long id);
```
Comment (padu12) [inline src/main/java/com/example/onlinebookstore/service/BookService.java:12]:
```suggestion
    BookDto getById(Long id);
```

**Review by padu12 — APPROVED** [review]
LGTM!

File: src/main/java/com/example/onlinebookstore/service/BookService.java (hunk 1 above)
Comment (Zel1oy) [reply src/main/java/com/example/onlinebookstore/service/BookService.java:12]: I changed it in my next PR

**Review by vovasalyha — APPROVED** [review]
GJ

**Pull request merged**